x-pack/legacy/plugins/siem/public/components/timeline/timeline.tsx

```diff
@@ -136,75 +123,57 @@ export const TimelineComponent: React.FC<Props> = ({
   const columnsHeader = isEmpty(columns) ? defaultHeaders : columns;
 
   return (
-    <TimelineContainer
-      data-test-subj="timeline"
-      direction="column"
-      gutterSize="none"
-      justifyContent="flexStart"
-    >
-      <WrappedByAutoSizer ref={measureRef as React.RefObject<HTMLDivElement>}>
-        <TimelineHeader
-          browserFields={browserFields}
-          id={id}
-          indexPattern={indexPattern}
-          dataProviders={dataProviders}
-          onChangeDataProviderKqlQuery={onChangeDataProviderKqlQuery}
-          onChangeDroppableAndProvider={onChangeDroppableAndProvider}
-          onDataProviderEdited={onDataProviderEdited}
-          onDataProviderRemoved={onDataProviderRemoved}
-          onToggleDataProviderEnabled={onToggleDataProviderEnabled}
-          onToggleDataProviderExcluded={onToggleDataProviderExcluded}
-          show={show}
-          showCallOutUnauthorizedMsg={showCallOutUnauthorizedMsg}
-          sort={sort}
-        />
-      </WrappedByAutoSizer>
+    <>
       <TimelineKqlFetch id={id} indexPattern={indexPattern} inputId="timeline" />
-      {combinedQueries != null ? (
```

Comment:
I'm seeing a change in behavior while desk testing (not necessarily related to this location) where the Timeline is populated by default, even on a fresh reload of the SIEM app.

As a result, `Create new timeline` appears to not work, because the timeline is populated with data even though there are no filters or KQL entered.

Reply:
Seeing the following in the JS console while desk testing (not necessarily related to this location) when clicking the `Load more` button:

```
react_devtools_backend.js:6 Warning: Encountered two children with the same key, `j2xwvXABOOUskGlPa61E`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
    in div (created by EventsTbody)
    in EventsTbody (created by EventsComponent)
    in EventsComponent
    in div (created by EventsTable)
    in EventsTable
    in div (created by TimelineBody)
    in TimelineBody
    in Unknown
    in Unknown
    in Unknown (created by ConnectFunction)
    in ConnectFunction (created by Query)
    in div (created by EuiFlyoutBody)
    in div (created by EuiFlyoutBody)
    in div (created by EuiFlyoutBody)
    in EuiFlyoutBody (created by StyledEuiFlyoutBody)
    in StyledEuiFlyoutBody (created by Query)
    in ManageTimelineContextComponent (created by Query)
    in Query (created by TimelineQueryComponent)
    in TimelineQueryComponent (created by EnhancedType)
    in EnhancedType (created by ConnectFunction)
    in ConnectFunction (created by TimelineComponent)
    in div (created by TimelineContainer)
    in TimelineContainer (created by TimelineComponent)
    in TimelineComponent (created by Query)
    in Query
    in Unknown
    in Unknown
    in Unknown (created by ConnectFunction)
    in ConnectFunction
    in div (created by Resizable)
    in Resizable (created by StyledResizable)
    in StyledResizable (created by FlyoutPaneComponent)
    in div (created by EuiFlyout)
    in div (created by FocusLock)
    in FocusLock (created by EuiFocusTrap)
    in div (created by OutsideEventDetector)
    in OutsideEventDetector (created by EuiFocusTrap)
    in EuiOutsideClickDetector (created by EuiFocusTrap)
    in EuiFocusTrap (created by EuiFlyout)
    in EuiFlyout (created by FlyoutPaneComponent)
    in div (created by EuiFlyoutContainer)
    in EuiFlyoutContainer (created by FlyoutPaneComponent)
    in FlyoutPaneComponent
    in div (created by Visible)
    in Visible
    in Unknown (created by ConnectFunction)
    in ConnectFunction (created by Query)
    in Provider (created by App)
    in App (created by ErrorBoundary)
    in ErrorBoundary (created by DragDropContext)
    in DragDropContext
    in Unknown
    in Unknown (created by ConnectFunction)
    in ConnectFunction (created by Query)
    in Query
    in Unknown (created by HomePage)
    in main (created by Main)
    in Main (created by HomePage)
    in div (created by WrappedByAutoSizer)
    in WrappedByAutoSizer (created by HomePage)
    in HomePage (created by PageRouterComponent)
    in Route (created by PageRouterComponent)
    in Switch (created by PageRouterComponent)
    in Router (created by PageRouterComponent)
    in ManageRoutesSpyComponent (created by PageRouterComponent)
    in PageRouterComponent (created by AppPluginRootComponent)
    in Unknown (created by AppPluginRootComponent)
    in ThemeProvider (created by AppPluginRootComponent)
    in ApolloProvider (created by AppPluginRootComponent)
    in Provider (created by AppPluginRootComponent)
    in ManageGlobalToaster (created by AppPluginRootComponent)
    in AppPluginRootComponent (created by StartAppComponent)
    in EuiContext (created by I18nContext)
    in PseudoLocaleWrapper (created by I18nProvider)
    in IntlProvider (created by I18nProvider)
    in I18nProvider (created by I18nContext)
    in I18nContext (created by StartAppComponent)
    in EuiErrorBoundary (created by StartAppComponent)
    in StartAppComponent (created by SiemAppComponent)
    in Provider (created by SiemAppComponent)
    in SiemAppComponent
```

Reply:
Seeing an issue (not necessarily related to this location) where items dragged from the timeline "fall behind" the header, per the following screenshot:

![falls-behind-while-dragging](https://user-images.githubusercontent.com/4459398/76253253-f56fd580-620f-11ea-8e2b-e98b2a996e94.gif)

Reply:
Fields in expanded events also fall behind, per the following gif:
![expanded-fields-fall-behind](https://user-images.githubusercontent.com/4459398/76253589-9494cd00-6210-11ea-93e2-20e7678cb514.gif)

Reply:
Seeing an issue where the columns in the Events table cannot be re-ordered, per the following gif:

![cannot-re-order-events-columns](https://user-images.githubusercontent.com/4459398/76257133-b5f8b780-6216-11ea-8e26-475173359dfb.gif)

Reply:
Seeing an issue in the Events table where a column cannot be added via D&D from an expanded event, per the following gif:

![events-cannot-drag-column-from-expanded-event](https://user-images.githubusercontent.com/4459398/76257583-92823c80-6217-11ea-8179-993cdd466ece.gif)